metadata-ingestion/docs/sources/snowflake/snowflake_pre.md

```diff
@@ -35,6 +35,9 @@ create user datahub_user display_name = 'DataHub' password='' default_role = dat
 
 // Grant the datahub_role to the new DataHub user.
 grant role datahub_role to user datahub_user;
+
+// Optional - required if extracting lineage, usage or tags (without lineage)
+grant imported privileges on database snowflake to role datahub_role;
```

Comment:
This is added here so that its present in the block containing other permissions. Earlier this was only present in the later sections after a lot of text and it was observed that this permission gets missed leading to missing lineage after ingestion.

Reply:
thanks - this is super helpful